anvil/src/cmd.rs

```diff
@@ -411,6 +412,12 @@ pub struct AnvilEvmArgs {
     #[clap(long, requires = "fork_url", help_heading = "Fork config")]
     pub no_storage_caching: bool,
 
+    /// Specify whether to be strict in json rpc handling of the upstream provider
```

Comment:
This docs are good—let's add just a tad more context as to _why_ we're doing this, talk a little bit about hardhat's node handling and such.